apps/i18n/common/en_us.json

```diff
@@ -1131,6 +1131,7 @@
   "recommended": "Recommended",
   "recommendedBlockContextualHintTitle": "Try using a block like this to solve the puzzle.",
   "record": "Record",
+  "recordedSoundsBrowserWarning": "Warning: Recorded sounds may not play correctly on mobile phones or other browsers. If you want to use a recorded sound on your phone, try uploading it from your computer.",
```

Comment:
I'm not sure this second sentence is clear - i.e. "What are you using the recorded sound for?". Perhaps something like "If you want to ensure the sound works on mobile phones, . . ."?

Reply:
@ryansloan can you weigh in on the wording of this warning?

Reply:
How about "Warning: Recorded sounds may not play correctly on mobile phones or other browsers. If you want to use your app on your phone, try uploading a sound from your computer instead."